run.sh

```diff
@@ -2,10 +2,18 @@
 set -e
 
 # Start the database
-if [[ -z "${DJANGO_DATABASE_URL}" ]]; then
+#
+if [ -z "${POSTGRES_USER}" ] || [ -z "${POSTGRES_PASSWORD}" ] \
+  || [ -z "${POSTGRES_HOST}" ] || [ -z "${POSTGRES_PORT}" ] \
+  || [ -z "${POSTGRES_DB}" ]
+then
   echo "Starting inbuilt database"
   ./db-run.sh
-  export DJANGO_DATABASE_URL='postgres://postgres:mathesar@localhost:5432/mathesar_django'
+  export POSTGRES_USER=postgres
+  export POSTGRES_PASSWORD=mathesar
+  export POSTGRES_HOST=localhost
+  export POSTGRES_PORT=5432
+  export POSTGRES_DB=mathesar_django
   if [[ -z "${MATHESAR_DATABASES}" ]]; then
```

Comment:
@mathemancer this script fails when all the `POSTGRES_*` variables are explicitly provided but `MATHESAR_DATABASES` isn't.

Here is the command I ran:
```bash
docker run \
-e POSTGRES_DB=mathesar_django \
-e POSTGRES_HOST=localhost \
-e POSTGRES_PORT=5432 \
-e POSTGRES_USER=mathesar \
-e POSTGRES_PASSWORD='mathesar' \
-v static:/code/static \
-v media:/code/media \
-v postgresql_config:/etc/postgresql/ \
-v postgresql_data:/var/lib/postgresql/ \
--name mathesar_service  \
-p 8000:8000 \
--restart unless-stopped \
mathesar/mathesar-prod:latest
```

Reply:
Good catch; I've pushed a commit to fix it.

Reply:
Works perfectly after the fix, Brent!